src/js/puzzles.js

```diff
@@ -314,10 +314,29 @@ var puzzlesArray = [
   word: 'cards',
   hint: 'unit of distance',
   answer: 'yards'
+},
+{
+  id: 46,
+  sentence: 'I left my boat at the rock',
+  word: 'rock',
+  hint: 'on water',
```

Comment:
Can we get a slightly better hint? Perhaps something like "building on water" or "building for ships".

Reply:
About to send pr with changes